src/envo/envo-edit.owl

```diff
@@ -40466,6 +40467,17 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105125> "plastic manufacturing process"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105125> <http://purl.obolibrary.org/obo/ENVO_01000993>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105126> (weathering of plastic)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105126> "Weathering during which plastics are worn or broken down."@en)
```

Comment:
Better to keep it singular (otherwise you constrain this term to only apply when you have multiple plastics)

> Weathering during which plastic is worn or broken down.

Reply:
Yes, it's also a mass noun (like water or soil)